### ISSUE TITLE
lfs track should identify already-tracked files via globs

### ISSUE THREAD

**Issue by cscrimge** [git-lfs/git-lfs#1919]
If I have already added the following rule to `.gitattributes`:
```*.tar.gz filter=lfs diff=lfs merge=lfs -text```
Then the command `lfs track path/to/file.tar.gz` should treat `path/to/file.tar.gz` as already tracked, and not add it to `.gitattributes`

**Comment by technoweenie** [comment]
This is a good idea... but why are you running `lfs track path/to/file.tar.gz` at all? Adding those patterns to `.gitattributes` is its sole purpose.

**Comment by cscrimge** [comment]
I agree that it's something I shouldn't need to do... but sometimes I forget. Or I added *.tar.gz, and someone else runs `lfs track path/to/file.tar.gz`.
